test_nx/nx_g.e2_error_parallel_edge

```diff
@@ -0,0 +1,4 @@
+src,dst
```

Comment:
what's the name `e2_error_parallel_edge` indicating?

Reply:
gs graph should be loaded without parallel edges.
```
        # graph have parallel edge in two edge label
        # load a graph contain parallel edges in 'e1' and  'e2'
        gs_g = load_from(vertices={
            'v1': ([("id", VId.INT)], loader(vfile1)),
            'v2': ([("id", VId.INT)], loader(vfile2)),
            'v3': ([("id", VId.INT)], loader(vfile3)),
            'v4': ([("id", VId.INT)], loader(vfile4))
        },
                       edges={
                           'e1': ('v1', 'v2', 'both_out_in', [
                               ("src", SrcId.INT), ("dst", DstId.INT)
                           ], loader(efile)),
                           'e2':
                           ('v3', 'v4', 'both_out_in', [("src", SrcId.INT),
                                                        ("dst", DstId.INT)],
                            loader(efile))
                       })
        with pytest.raises(RuntimeError):
            nx_g = self.Graph(gs_g)
```
It seems that this part should be improved. I'll check this part with weibin